github/lib/configure_repo.rb

```diff
@@ -101,6 +101,16 @@ def required_status_checks
              .fetch("additional_contexts", [])
         ].compact
       }
+    elsif github_actions_exists?
```

Comment:
Does this have to be a mutually exclusive action? Or should it be an `if` rather than an `elsif`?

Reply:
Jenkins and GitHub Actions are our two supported CI providers. Yet it's perfectly valid for a repo to have no CI. It's also valid for a repo to run _CI_ with GitHub Actions and use Jenkins for gem releases (once [this govuk-jenkinslib setting PR is merged](https://github.com/alphagov/govuk-jenkinslib/pull/70)). Hmm. Good question. I'm not sure!

Reply:
Hmm, and there are some repos that have both (`static`, `collections`) due to having been taken private and having had GitHub Actions added to them when private. Now I'm even more confused about the intended behaviour here. I _think_ we want:

- Configure Jenkins when a Jenkinsfile is present.
- Configure Actions when `.github/workflows/ci.yml` is present.
- Don't configure anything if neither of them are present.

But I'm not sure what to do when _both_ are present.

Reply:
The consensus from floating this "what do we do when a repo has both Actions and Jenkins" question in Slack is that we should configure them both. I'll try and figure out how to do this.

Reply:
I was thinking something like this:

```ruby
  def required_status_checks
    if jenkinsfile_exists? || github_actions_exists?
      {
        strict: overrides.fetch("up_to_date_branches", false),
        contexts: [
          jenkinsfile_exists? ? "continuous-integration/jenkins/branch" : nil,
          jenkinsfile_runs_e2e_tests? ? "continuous-integration/jenkins/publishing-e2e-tests" : nil,
          github_actions_exists? ? "test" : nil,
          *overrides
              .fetch("required_status_checks", {})
              .fetch("additional_contexts", [])
        ].compact
      }
    end
  end
```

But the test stubs are annoyingly stubborn.

Reply:
Solved the last failing test with @steveholmes' help - thanks Steve!